Sources/Plasma/PubUtilLib/plNetMessage/plNetMessage.h

```diff
@@ -44,6 +44,7 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 
 #include "HeadSpin.h"
 #include "hsBitVector.h"
+#include "hsStream.h"
```

Comment:
The include-graph mindful part of me would like to see this moved into the source file if it would be a trivial change.